Entity Explorer - Domain and URL.ipynb

```diff
@@ -87,30 +87,30 @@
    },
```

Comment:
<p>I did change the version numbers to be text strings in the other notebooks but not a big deal - it works with both</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>